### ISSUE TITLE
FR: Implement ImportUsersAsync() API

### ISSUE THREAD

**Issue by hiranya911** [firebase/firebase-admin-dotnet#185]
Implement an API for importing Firebase Auth user accounts in bulk. Partial API proposal:

```
class FirebaseAuth
    Task<UserImportResult> ImportUsersAsync(IEnumerable<ImportUserRecordArgs> users)
    Task<UserImportResult> ImportUsersAsync(IEnumerable<ImportUserRecordArgs> users, UserImportOptions options)

class ImportUserRecordArgs
    string Uid { get; set; }
    string Email { get; set; }
    boolean EmailVerified { get; set; }
    string DisplayName { get; set; }
    string PhoneNumber { get; set; }
    string PhotoUrl { get; set; }
    boolean Disabled { get; set; }
    UserMetadata UserMetadata { get; set; }
    byte[] PasswordHash { get; set; }
    byte[] PasswordSalt { get; set; }
    IEnumerable<UserProvider> UserProviders { get; set; }
    IReadOnlyDictionary<string, object> CustomClaims  { get; set; }

class UserImportResult
    int SuccessCount { get; }
    int FailureCount { get; }
    IReadOnlyList<ErrorInfo> Errors { get; }
```

**Comment by MathBunny** [comment]
Merged in #191, remaining work is to make a release that includes the change.

**Comment by hiranya911** [comment]
Will be included in the next release.
